lib/datadog/kit/events.rb

```diff
@@ -0,0 +1,31 @@
+# typed: false
+
+module Datadog
+  module Kit
+    # Tracking events
+    module Events
+      def self.track_login(trace, id:, success:, **others)
+        subtag = success ? 'success' : 'failure'
```

Comment:
I recommend adding the frozen_string_literal comment to all new production files.

Reply:
Good point. I wonder why Rubocop didn't yell at me (I recall it usually does for this).

Reply:
Ah that's because of this:

```
# TODO: Disabling until we categorize which file are safe to do so.
Style/FrozenStringLiteralComment:
  Enabled: false
```